go/oasis-node/cmd/debug/txsource/workload/junk.go

```diff
@@ -0,0 +1,28 @@
+package workload
```

Comment:
What's with the `junk.go` filename? It currently doesn't do anything, but maybe call it something else or remove it?

Reply:
I'll rename it